packages/experiments/src/components/TilesList/TilesList.tsx

```diff
@@ -351,7 +380,10 @@ export class TilesList<TItem> extends React.Component<ITilesListProps<TItem>, IT
 
     const { cells } = this.state;
 
-    const endIndex = Math.min(cells.length, startIndex + CELLS_PER_PAGE);
+    const endIndex = Math.min(
+      cells.length,
+      this.props.cellsPerPage && this.props.cellsPerPage > 0 ? startIndex! + this.props.cellsPerPage : startIndex! + CELLS_PER_PAGE
```

Comment:
Is the non-null assertion necessary? It wasn't there prior.

Reply:
I removed the non null assertions.  We were running our code with different linter setting that required that to be there.